client/src/handlers/handlers.ts

```diff
@@ -260,3 +260,202 @@ export function makeHandleRunJobWithProgress (
     })
   }
 }
+
+/**
+ * creates a new project in the current directory using command `java -jar flix.jar init`
+ * 
+ * @param context vscode.ExtensionContext
+ * 
+ * @param launchOptions LaunchOptions
+ * 
+ * @returns function handler
+ */
+ export function cmdInit(
+    context: vscode.ExtensionContext, 
+    launchOptions: LaunchOptions = defaultLaunchOptions
+    ) {
+        return async function handler () {
+            const flixFilename = await getFlixFilename(context, launchOptions)
+            const cmd = ['java', '-jar', flixFilename, 'init']
+            let terminal = vscode.window.createTerminal({name: `init`})
+            passCommandToTerminal(cmd, terminal)
+        }
+}
+
+/**
+ * checks the current project for errors using command `java -jar flix.jar check`
+ * 
+ * @param context vscode.ExtensionContext
+ * 
+ * @param launchOptions LaunchOptions
+ * 
+ * @returns function handler
+ */
+
+export function cmdCheck(
+    context: vscode.ExtensionContext, 
+    launchOptions: LaunchOptions = defaultLaunchOptions
+    ) {
+        return async function handler () {
+            const flixFilename = await getFlixFilename(context, launchOptions)
+            const cmd = ['java', '-jar', flixFilename, 'check']
+            let terminal = vscode.window.createTerminal({name: `check`})
+            passCommandToTerminal(cmd, terminal)
+        }
+}
+
+/**
+ * builds (i.e. compiles) the current project using command `java -jar flix.jar build`
+ * 
+ * @param context vscode.ExtensionContext
+ * 
+ * @param launchOptions LaunchOptions
+ * 
+ * @returns function handler
+ */
+export function cmdBuild(
+    context: vscode.ExtensionContext, 
+    launchOptions: LaunchOptions = defaultLaunchOptions
+    ) {
+        return async function handler () {
+            const flixFilename = await getFlixFilename(context, launchOptions)
+            const cmd = ['java', '-jar', flixFilename, 'build']
+            let terminal = vscode.window.createTerminal({name: 'build'})
```

Comment:
I think creating a new terminal for each of these is too much.

Let us at least reuse the terminal with the same name (if it exists).